python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,103 @@
+
+args = parser.parse_args()
+FROM, TO, NUM_TXS = int(args.FROM), int(args.TO), int(args.NUM_TXS)
+
+if FROM == TO:
+    sys.exit('Error: The values of arguments "TO" and "FROM" are equal make sure to set different values')
+
+RPC, num_msgs = "http://127.0.0.1:16657", 30
+
+#### Fetching Bech32 addresses ######
+
+acc1, acc1err = fetch_account_address(f"account{FROM}")
+if len(acc1err):
+    sys.exit(acc1err)
+
+acc2, acc2err = fetch_account_address(f"account{TO}")
+if len(acc2err):
+    sys.exit(acc2err)
+
+#### Fetch Balances of acc1 acc2 before execting the load test ####
+before_acc1_balance, before_acc1_balanceerr = query_balances(acc1, RPC, amount = True)
+if len(before_acc1_balanceerr):
+    sys.exit(before_acc1_balanceerr)
+
+before_acc2_balance, before_acc2_balanceerr = query_balances(acc2, RPC, amount = True)
+if len(before_acc2_balanceerr):
+    sys.exit(before_acc2_balanceerr)
+
+#### Fetching sequence numbers of to and from accounts
+os.chdir(os.path.expanduser(HOME))
+status, seq1no = fetch_seq_no(acc1, RPC)
+if not status:
+    sys.exit(seq1no)
+
+status, seq2no = fetch_seq_no(acc2, RPC)
+if not status:
+    sys.exit(seq2no)
+
+#### Generating unsigned transactions with a single transfer message 
+for i in range(0, int(NUM_TXS)):
+    status, unsignedTxto = unsigned_tx(acc1, acc2, 'unsignedto.json')
+    if not status:
+        print(unsignedTxto)
+    
+    status, unsignedTxfrom = unsigned_tx(acc2, acc1, 'unsignedfrom.json')
+    if not status:
+        print(unsignedTxfrom)
+#### Duplicating and appending transfer message in the existing array to create a multi-msg transaction        
+    for j in range(0, int(num_msgs)):
+        write_json('unsignedto.json')
+        write_json('unsignedfrom.json')
+
+    ### Signing and broadcasting the unsigned transactions from acc1 to acc2 ###
+    seqto = seq1no + i
+    status, txHash = signed_tx('unsignedto.json', 'signedto.json', acc1, seqto, RPC)
+    if not status:
+        print(txHash)
+    else:
+        print(f"broadcasttoTxhash: {txHash}")
+
+    ### Signing and broadcasting the unsigned transactions from acc2 to acc1 ###
+    seqfrom = seq2no + i
+    status, txHash = signed_tx('unsignedfrom.json', 'signedfrom.json', acc2, seqfrom, RPC)
+    if not status:
+        print(txHash)
+    else:
+        print(f"broadcastfromTxhash: {txHash}")
+
+print('##### Sleeping for 7s #####')
+time.sleep(7)
+
+#### Verifying the balance deductions ####
+after_acc1_balance, after_acc1_balanceerr = query_balances(acc1, RPC, amount = True)
+if len(after_acc1_balanceerr):
+    sys.exit(after_acc1_balanceerr)
+
+after_acc2_balance, after_acc2_balanceerr = query_balances(acc2, RPC, amount = True)
+if len(after_acc2_balanceerr):
+    sys.exit(after_acc2_balanceerr)
+
+acc1_diff = int(before_acc1_balance) - int(after_acc1_balance)
+acc2_diff = int(before_acc2_balance) - int(after_acc2_balance)
+
+print_balance_deductions('account1', acc1_diff)
+print_balance_deductions('account2', acc2_diff)
```

Comment:
should we instead `validate_balances()`. Like after the txn successful, we should cross check the queried balances against calculated balances